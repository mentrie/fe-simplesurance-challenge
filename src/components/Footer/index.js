import React from 'react';

const Footer = () => (
  <footer className="footer">
    <p>Designed and built by <a href="https://twitter.com/mentrie" className="footer__twitter__link">@mentrie</a></p>
  </footer>
);

export default Footer;