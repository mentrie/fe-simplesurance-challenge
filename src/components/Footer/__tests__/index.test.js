import React from 'react';
import Footer from '../index';

test('It should render component', () => {
  const component = shallow(<Footer />);
  expect(component).toMatchSnapshot()
});

test('It should render a link tag', () => {
  const component = shallow(<Footer />);
  expect(component.find('a').length).toBe(1);
});

