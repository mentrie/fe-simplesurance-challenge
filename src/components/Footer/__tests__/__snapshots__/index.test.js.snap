// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render component 1`] = `
<footer
  className="footer"
>
  <p>
    Designed and built by 
    <a
      className="footer__twitter__link"
      href="https://twitter.com/mentrie"
    >
      @mentrie
    </a>
  </p>
</footer>
`;
